__tests__/app.test.js

```diff
@@ -56,4 +56,35 @@ describe('GET Categories', () => {
             });
         })
     });
+});
+
+describe('POST Comment', () => {
+    test('Should add comment to database', () => {
+
+        const payload = {
+            username: "philippaclaire9",
+            body: "This game ruined Christmas. My whole family hate me. 5 Stars."
+        }
+        return request(app).post('/api/reviews/13/comments')
+        .send(payload)
+        .expect(201)
+        .then((response) => {
+            const comment = response.body.comment;
+            expect(comment).toHaveProperty('comment_id');
+            expect(comment.body).toBe("This game ruined Christmas. My whole family hate me. 5 Stars.");
+            expect(comment.review_id).toBe(13);
+            expect(comment.author).toBe('philippaclaire9');
+            expect(comment.votes).toBe(0);
+            expect(comment).toHaveProperty('created_at');
+
+            return db.query(`
+            SELECT * FROM COMMENTS WHERE review_id = 13`)
+            .then((result) => {
+                const databaseComment = result.rows[0];
+                expect(databaseComment.review_id).toBe(comment.review_id)
+                expect(databaseComment.body).toBe(comment.body)
+                expect(databaseComment.comment_id).toBe(comment.comment_id)
+            })
+        })
```

Comment:
Not sure this is needed, as the response from the db tells you its there. 
Also, this is a nested .then -> which isnt ideal for best practice and all that.